internal/search/datasources/postgres/postgres.go

```diff
@@ -121,22 +121,9 @@ func makeSQL(index string, srid d.SRID, bboxFilter string) string {
 			r.bbox,
 			r.geometry,
 			r.suggest,
-			CASE WHEN r.display_name = r.suggest THEN
-				(
-				    ts_rank_cd(r.ts, (SELECT query FROM query_exact), $7) * $8 * $9 + ts_rank_cd(r.ts, (SELECT query FROM query_wildcard), $7)
-				) * rel.relevance
-			ELSE
-				(
-				    ts_rank_cd(r.ts, (SELECT query FROM query_exact), $7) * $8 + ts_rank_cd(r.ts, (SELECT query FROM query_wildcard), $7)
-				) * rel.relevance
-			END AS rank,
-			ts_headline('custom_dict', r.suggest, (SELECT query FROM query_wildcard)) AS highlighted_text
+			r.ts
 		FROM
 			%[1]s r
-		LEFT JOIN
-			(SELECT * FROM unnest($4::text[], $6::float[]) rel(collection_id,relevance)) rel
-		ON
-			rel.collection_id = r.collection_id
 		WHERE
 			r.ts @@ (SELECT query FROM query_wildcard) AND (r.collection_id, r.collection_version) IN (
 				-- make a virtual table by creating tuples from the provided arrays.
```

Comment:
Dit stukje kan misschien ook nog wel in een CTE:

```
WITH collection_id_and_version AS (
    SELECT collection_id, collection_version
    FROM unnest($4::text[], $5::int[])
)
```

En dan hier aanroepen via 
```
AND (r.collection_id, r.collection_version) IN (
        SELECT collection_id, collection_version FROM collection_id_and_version
    )
```